recipes/pulseaudio/all/conanfile.py

```diff
@@ -0,0 +1,133 @@
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_autotools(self):
+        if not self._autotools:
+            self._autotools = AutoToolsBuildEnvironment(self)
+            args=[]
+            for lib in ['alsa', 'x11', 'openssl', 'dbus']:
+                args.append("--%s-%s" % ('enable' if getattr(self.options, 'with_' + lib) else 'disable', lib))
+            args.append("--%s-glib2" % ('enable' if self.options.with_glib else 'disable'))
+            args.append("--%s-fftw" % ('with' if self.options.with_fftw else 'without'))
+            if self.options.shared:
+                args.extend(['--enable-shared=yes', '--enable-static=no'])
+            else:
+                args.extend(['--enable-shared=no', '--enable-static=yes'])
+            args.append("--with-udev-rules-dir=%s" % os.path.join(self.package_folder, "bin", "udev", "rules.d"))
+            with tools.environment_append({"PKG_CONFIG_PATH": self.build_folder}):
+                with tools.environment_append({
+                        "FFTW_CFLAGS": tools.PkgConfig("fftw").cflags,
+                        "FFTW_LIBS": tools.PkgConfig("fftw").libs}):
+                    with tools.environment_append(RunEnvironment(self).vars):
+                        self._autotools.configure(args=args,  configure_dir=self._source_subfolder)
+        return self._autotools
+
+    def build(self):
+        
+        for package in self.deps_cpp_info.deps:
+            lib_path = self.deps_cpp_info[package].rootpath
+            for dirpath, _, filenames in os.walk(lib_path):
+                for filename in filenames:
+                    if filename.endswith('.pc'):
+                        shutil.copyfile(os.path.join(dirpath, filename), filename)
+                        tools.replace_prefix_in_pc_file(filename, lib_path)
+
+        autotools = self._configure_autotools()
+        autotools.make()
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        autotools = self._configure_autotools()
+        autotools.install()
+        tools.rmdir(os.path.join(self.package_folder, 'etc'))
+        tools.rmdir(os.path.join(self.package_folder, 'share'))
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+        for f in glob.glob(os.path.join(self.package_folder, "lib", "**", "*.la"), recursive=True):
+            os.remove(f)
+
+    def package_info(self):
+        self.cpp_info.libdirs = ['lib', os.path.join('lib', 'pulseaudio')]
+        if self.options.with_glib:
+            self.cpp_info.libs.append('pulse-mainloop-glib')
+        self.cpp_info.libs.extend(['pulse-simple', 'pulse'])
+        if not self.options.shared:
+            self.cpp_info.libs.append('pulsecommon-%s' % self.version)
+        if self.options.with_dbus:
+            self.cpp_info.system_libs.append('dbus-1')
```

Comment:
That is kinda a cheat? Shouldn't a dbus package be used?

Reply:
You caught me :D (For my defense, I think there was no libusb package at the time I originally made this PR)
I'll fox it later today

Reply:
You cant wait for #2147 or remove that option and add a TODO.